### ISSUE TITLE
res2net50移植到faster-rcnn网络中，检测结果不好

### ISSUE THREAD

**Issue by StrongBirds** [gasvn/Res2Net#15]
作者你好，非常感谢你的代码，我将pytorch-faster-rcnn的特征提取网络换成了res2net50_26w_4s，在VOC2007上训练和测试，最终的测试结果mAP = 71.05%,没有得到你论文中的mAP=74.4%，请问是什么原因呢？我将res2net50_26w_4s的block1设置为不参加训练，block4设置stype = 'normal'，使得在block4中的stride=1，请问我有哪些需要注意的地方吗？我用的faster-rcnn源代码是pytorch-faster-rcnn,期待你的回复

**Comment by gasvn** [comment]
Thanks for your interest.  Make sure that you can reproduce the results of baseline.  
Please refer to the detection code of Res2Net: https://github.com/gasvn/Res2Net-maskrcnn

**Comment by StrongBirds** [comment]
我今天使用相同的pytorch-faster-rcnn代码，仅仅将特征提取网络换成了resnet50得到mAP=74.4%，resnet101是mAP = 75.36%。 res2net50_26w_4s得到是mAP = 71.05%。我在看看res2net-maskrcnn去找找我的问题所在，感谢你的回复。

**Comment by gasvn** [comment]
Is the pre-trained model of resnet50 from torchvision? Please be careful that the input image should be normalized as follows:
normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                  std=[0.229, 0.224, 0.225])

**Comment by StrongBirds** [comment]
我使用的预训练模型来自于这个作者https://github.com/ruotianluo/pytorch-resnet，这与torchvision中的应该没啥大区别吧，恳请你的指教。

**Comment by gasvn** [comment]
Please try the pretrained model from torchvision to train the baseline. Res2Net shares the same pre-processing method with models from torchvision.
Models trained with caffe and pytorch usually use differenet pre-processing methods.

**Comment by StrongBirds** [comment]
好的，谢谢，我在试一试
